apps/azure-functions-api/src/domain/openid-federation.ts

```diff
@@ -0,0 +1,46 @@
+import * as RTE from 'fp-ts/lib/ReaderTaskEither';
+import { getJwkPublicKeyList, signJwt } from './signer';
+import { pipe } from 'fp-ts/lib/function';
+
+// TODO: Move to configuration
+const baseURL = 'https://io-d-itn-eudiw-api-func-01.azurewebsites.net';
+
+export const getFederationMetadata = pipe(
+  RTE.Do,
+  RTE.apSW('jwks', getJwkPublicKeyList),
+  RTE.map(({ jwks }) => ({
+    iss: baseURL,
+    jwks: {
+      keys: jwks,
+    },
+    metadata: {
+      federation_entity: {
+        homepage_uri: 'https://io.italia.it',
+        logo_uri: 'https://io.italia.it/assets/img/io-it-logo-blue.svg',
+        organization_name: 'PagoPa S.p.A.',
+        policy_uri: 'https://io.italia.it/privacy-policy',
+        tos_uri: 'https://io.italia.it/privacy-policy',
```

Comment:
**note**: I have hardcoded it for now, but we will move them to the configuration later